backend/app/services/conversation_scenario_service.py

```diff
@@ -184,23 +191,48 @@ def list_scenarios_summary(
                 ConversationCategory.name,
                 ConversationScenario.custom_category_label,
             )
+            .order_by(col(ConversationScenario.created_at).desc())
```

Comment:
Can we order these by most recent session? the scenario with the most recent session should be returned first.